starter-code/server.js

```diff
@@ -56,8 +58,12 @@ app.post('/articles', function(request, response) {
 
   function queryThree(author_id) {
     client.query(
-      ``, // TODO: Write a SQL query to insert the new article using the author_id from our previous query
-      [], // TODO: Add the data from our new article, including the author_id, as data for the SQL query.
+      `INSERT INTO articles
+      (author_id, title, category, "publishedOn", body)
+      VALUES
+      (${author_id},$1,$2,$3,$4)
```

Comment:
As discussed in class, it's generally best to avoid string interpolation even when it's technically safe, to take advantage of [Plan Caching](https://www.postgresql.org/docs/9.5/static/plpgsql-implementation.html#PLPGSQL-PLAN-CACHING).